scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -52,6 +52,50 @@
 - name: DAC_OUT
   match: "^DAC(?:\\d+)?_OUT\\d+$"
 
+- name: DCMI_HSYNC
+  match: "^DCMI_HSYNC$"
+  slew-rate: very-high-speed
+
+- name: DCMI_PIXCLK
+  match: "^DCMI_PIXCLK$"
+  slew-rate: very-high-speed
+
+- name: DCMI_VSYNC
+  match: "^DCMI_VSYNC$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D0
+  match: "^DCMI_D0$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D1
+  match: "^DCMI_D1$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D2
+  match: "^DCMI_D2$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D3
+  match: "^DCMI_D3$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D4
+  match: "^DCMI_D4$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D5
+  match: "^DCMI_D5$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D6
+  match: "^DCMI_D6$"
+  slew-rate: very-high-speed
+
+- name: DCMI_D7
+  match: "^DCMI_D7$"
+  slew-rate: very-high-speed
```

Comment:
Could you use a regular expression ?
See for instance:
```
- name: FMC
  match: "^FMC_(?:NL|NADV|CLK|NBL[0-3]|A\\d+|D\\d+|NE[1-4]|NOE|NWE|NWAIT|NCE|INT|SDCLK|SDNWE|SDCKE[0-1]|SDNE[0-1]|SDNRAS|SDNCAS)$"
```
Aside from conciseness, this is slightly faster to generate.

Reply:
Done